subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testng/TestNGJdkNavigationTest.groovy

```diff
@@ -22,7 +37,7 @@ class TestNGJdkNavigationTest extends AbstractIntegrationTest {
     void shouldNotNavigateIntoJdkClasses() {
         executer.withTasks('test').run()
 
-        new DefaultTestExecutionResult(testDirectory).testClass('org.grade.Test').assertTestPassed('shouldPass')
+        new DefaultTestExecutionResult(testDirectory).testClass('org.grade.Test1').assertTestPassed('shouldPass')
     }
```

Comment:
Please fix this type: `'org.grade.Test1'` should be `'org.gradle.Test1'` (test is currently failing because of that)